styles/about.css

```diff
@@ -48,3 +48,27 @@ body {
   color: #000;
   margin-top: 30px;
 }
+
+main {
+  margin: 40px auto 48px;
+  max-width: max-content;
+}
+
+main > h2 {
+  font-size: 40px;
+  line-hight: 48px;
+  margin-bottom: 40px;
+}
+
+.content {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 130px;
+}
+
+.content article {
+  font-size: 24px;
+  line-height: 29px;
+  max-width: 640px;
```

Comment:
@RafaelMariscal , for the article container, It should have a margin right and left of 70px to align with the navbar. Check it out on Figma Design.

Reply:
Ok! Applying changes right now!

Reply:
Good job!